src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorControllerReplaceIGMTest.java

```diff
@@ -66,6 +66,12 @@
     @MockBean
     private NetworkConversionService networkConversionService;
 
+    @MockBean
+    private MergeOrchestratorConfigService mergeOrchestratorConfigService;
+
+    @MockBean
+    private MergeEventService mergeEventService;
```

Comment:
why add this Mock service ?

Reply:
We need to add these Mock services here, in order to be able to run the test (the inheritance with AbstractEmbeddedCassandraSetup has been removed)